lib/utf8.pm

```diff
@@ -1,15 +1,19 @@
 package utf8;
 
-$utf8::hint_bits = 0x00800000;
```

Comment:
I'm aware this might have unexpected side effects but I wasn't able to find any. I'm happy to revert this if there's a good reason for not allowing strict/warnings here. Is this a circular deps issue maybe?

Reply:
I don't know if this has side effects or not.

BTW I hate that we don't derive the bit number.

Reply:
So, apply it and see what breaks on cpan, hopefully nothing

Reply:
I can't remember, but isn't this global the way it tells the core that 'use utf8' is in effect?  If so, this would break lots.

Reply:
I don't know about possibility of circular reference, but applying strict and warnings to this module could be a significant performance hit due to how much it is used. IMO making the other changes and leaving out strict and warnings would be preferable in this specific case. By the way, bytes.pm is a "mirror" of this module so should be updated in the same way.

Reply:
This global is still being set a few lines below.

Reply:
I hear what you're saying. I hesitated for similar reasons in all honesty. 

 - require is almost no cost if the module is already loaded. This would only happen the first time utf8.pm is loaded anyway.
 - strict/warnings->import would also only happen once. Why would that be a performance hit?

Reply:
I'm not talking about the performance hit of activating strict/warnings but of having them active each time this module is imported or its AUTOLOAD is called. Though there are certainly bigger concerns at that point. I certainly don't think it will make a big difference but just questioning the tradeoff for this module in particular, maybe that's why they weren't set before, maybe not.